fachhochschule-vorarlberg-author-date.csl

```diff
@@ -92,54 +96,39 @@
   </macro>
   <macro name="editor">
     <choose>
-      <if variable="author">
-        <choose>
-          <if type="chapter">
-            <names variable="editor" delimiter="; ">
-              <name name-as-sort-order="all" sort-separator=", " delimiter="; " delimiter-precedes-last="never" form="long" suffix=""/>
-              <label form="short" prefix=" (" suffix=".): "/>
-            </names>
-          </if>
-        </choose>
-      </if>
+      <if variable="author"></if>
       <else>
         <names variable="editor" delimiter="; ">
           <name name-as-sort-order="all" sort-separator=", " delimiter="; " delimiter-precedes-last="never" form="long" suffix=""/>
-          <label form="short" prefix=" (" suffix=".) "/>
+          <label form="short" prefix=" (" suffix=".): "/>
```

Comment:
@cookie365, it's better to move the ": " in the suffix up to the `<names/>` element.